flutter/news/lib/core/network/network_info.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:internet_connection_checker/internet_connection_checker.dart';
+
+abstract class NetworkInfo {
+  Future<bool> get isConnected;
+}
+
+class NetworkInfoImpl implements NetworkInfo {
```

Comment:
If we only have an implementation of `NetworkInfo`... do we need the interface? (abstract class)

Reply:
updated on https://github.com/novoda/multiplatform-playground/commit/8323a52c3c021a85d1662fa2609c061534a5fce3